pyproject.toml

```diff
@@ -44,6 +44,9 @@ dependencies = [
   "PyYAML",
   "sdmx1 >= 2.13.1",
   "tqdm",
+  "pyxlsb >= 1.0.10",
+  "scikit-learn",
+  "nomenclature-iamc",
```

Comment:
@measrainsey is `nomenclature-iamc` actually used in the code? I can't seem to find where.